command_line/rescale_diamond_anvil_cell.py

```diff
@@ -0,0 +1,302 @@
+# coding: utf-8
```

Comment:
Here for convenience - why is this called "rescale diamond anvil cell" rather than e.g. "correct diamond anvil" - purely a bikeshed thing but probably good to get right from the start.

Reply:
Yep, it's a good point.  Opinions, @jbeilstenedmands, @rjgildea?

Reply:
"correct" is potentially ambiguous in that it can be an adjective or a verb, whereas at least "rescale" is unambiguous in that sense...

Reply:
I've settled on `dials.anvil_correction`, I hope that appeases the bikehed committee?